include/h2o/http3_common.h

```diff
@@ -119,10 +119,21 @@ size_t h2o_http3_goaway_frame_capacity(quicly_stream_id_t stream_or_push_id);
 uint8_t *h2o_http3_encode_goaway_frame(uint8_t *buff, quicly_stream_id_t stream_or_push_id);
 int h2o_http3_decode_goaway_frame(h2o_http3_goaway_frame_t *frame, const uint8_t *payload, size_t len, const char **err_desc);
 
+/*
+ * Special dummy values for representing errors from http3 acceptor
+ */
+extern h2o_quic_conn_t h2o_quic_accept_conn_decryption_failed;
+extern h2o_http3_conn_t h2o_quic_accept_conn_closed;
```

Comment:
* Can we add `const`? We only use these values for testing addresses.
* Re `h2o_quic_accept_conn_closed`, I think we should rename it to `h2o_http3_accept_conn_closed`. It is a value specific to HTTP/3 (consider adding support for another application protocol on top of `h2o_quic_conn_t`).

Reply:
Turns out applying `const` isn't a trivial thing. Addresses of these variables will be returned from `h2o_http3_server_accept` but its return value is of type `h2o_http3_conn_t *` (not `const *`), so the compiler will not be happy. We can always cast to a non-const pointer to keep the compiler silent but it does not make too much sense.

OTOH I agree with the naming change.